app/controllers/sidekiq_monitoring/status_controller.rb

```diff
@@ -18,12 +18,18 @@ def status_data
 
     def timestamps
       {
-        whenever_ran: Redis.current.get('monitoring:timestamp:whenever_ran'),
-        sidekiq_performed: Redis.current.get('monitoring:timestamp:sidekiq_performed'),
+        whenever_ran: redis_get('monitoring:timestamp:whenever_ran'),
+        sidekiq_performed: redis_get('monitoring:timestamp:sidekiq_performed'),
         requested: Time.current.to_s(:db)
       }
     end
 
+    def redis_get(key)
+      Sidekiq.redis do |conn|
+        conn.get(key)
+      end
```

Comment:
i'd do both gets with the same conn object. Otherwise sidekiq might open up an extra connection in the pool.